main.py

```diff
@@ -350,6 +353,8 @@ def __add_zap_options__(self, args: List[str]) -> None:
             zap_options.append(self.__add_custom_option__(label="oauth.token_url", value=self.oauth_token_url))
         if self.oauth_parameters is not None:
             zap_options.append(self.__add_custom_option__(label="oauth.parameters", value=self.oauth_parameters))
+        if self.disable_rules is not None:
```

Comment:
check for a cleaner way to pass params to the hook

Reply:
done, this way os passing parameters was causing https://github.com/soos-io/soos-dast/pull/64